services/scripts/testy-snapshot.sh

```diff
@@ -71,10 +71,15 @@ main() {
 
     # Validate database. Update the snapshot status on success/failure.
     echo "Running validation script '$_validation_script' on volume '$_backup_volume_id'."
+    var=$(date +%s%3N)
```

Comment:
Is there any reason why this variable doesn't follow the convention of the others using `local` and prefixing with an `_`?

Reply:
var is used in bash to store temporary variables, so I have just followed that here. 

We are using var to temporarily save the timestamp output from bash, and it always gets used right after it is updated, and won't be used anywhere else and becomes redundant immediately after.

Reply:
Okay, I see, is it necessary then?

Reply:
Yup! I have to call it and save it somewhere before sending it into the logs. I cant call the bash command within the aws command unfortunately - or there may be a work around that I have not found.

Reply:
Okay, that's fine then!